services/node-api-service/implementations/http-service.js

```diff
@@ -1,17 +1,29 @@
 import axios from 'axios';
-import { OPERATION_STATUSES  } from '../../../constants.js';
-import { sleepForMilliseconds  } from '../../utilities.js';
+import { OPERATION_STATUSES } from '../../../constants.js';
+import { sleepForMilliseconds } from '../../utilities.js';
 
 export default class HttpService {
     constructor(config = {}) {
         this.config = config;
+
+        if (
+            !(
+                config.nodeApiVersion === '/' ||
+                config.nodeApiVersion === '/latest' ||
+                /^\/v\d+$/.test(config.nodeApiVersion)
+            )
+        ) {
+            throw Error(`Version must be '/latest', '/' or in '/v{digits}' format.`);
```

Comment:
https://github.com/OriginTrail/dkg.js/pull/190 private public support is here, remove it from this PR